CHANGELOG.md

```diff
@@ -41,7 +52,7 @@ stages](https://github.com/kubernetes/community/blob/master/contributors/devel/s
 - Reduce memory usage of Antrea Controller when an Agent establishes a connection.
 - Clean-up the appropriate iptables rules when a Node leaves the cluster.
 
-## 0.4.1 - 2019-02-27
```

Comment:
whoops

Reply:
can't believe I copy pasted the line 4 times and never noticed...